rllib/algorithms/algorithm_config.py

```diff
@@ -421,6 +421,8 @@ def __init__(self, algo_class=None):
         self._enable_rl_module_api = False
         # Whether to error out if exploration config is set when using RLModules.
         self._validate_exploration_conf_and_rl_modules = True
+        # Helper to keep track of the original exploration config.
+        self.__prior_exploration_config = None
```

Comment:
We ran into problems here since on might switch off/on RLModules twice or even more times.
This field simply backs up a previous exploration config.